src/core/evaluator.js

```diff
@@ -1204,7 +1205,7 @@ var PartialEvaluator = (function PartialEvaluatorClosure() {
 
       // The xobj is parsed iff it's needed, e.g. if there is a `DO` cmd.
       var xobjs = null;
-      var xobjsCache = Object.create(null);
+      var skipNonFormXobjs = Object.create(null);
```

Comment:
Based on the latest round of comments in https://github.com/mozilla/pdf.js/pull/8488#discussion_r120747114, the initially suggested name `skipEmptyXObjs` probably makes a lot more sense.

Again, sorry about causing unnecessary confusion here!